### PR TITLE
Archiving byzcoin

### PR DESCRIPTION
**Pull request by ineiti**
Archiving of the byzcoin service:
- creating docker images with a pre-initialized blockchain
- include omniledger, ol-explorer, and columbus-united as webapps
- add a landing page with explanations

Code review steps:
- follow README.md to start the demo
- go through the signup and test the other two demos
- give feedback/create PRs on:
  - text itself (grammar, understandability)
  - how easy it is to follow the instructions
  - the description of the `Troubleshooting` part
- regarding the code, only point out big mistakes that really should be cleaned up before merging. I don't expect you to go through all files added / modified, especially not those in `pkg*`, which are mostly copied from `cothority`.

Closes c4dt/TODO#324

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Dockerfile*
+docker-compose*
```
Comment (PascalinDe) [inline .dockerignore:2]:
```suggestion
docker-compose*
.github
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+RUN npm i
+RUN npm run bundle
+COPY --from=builder-byzcoin /byzcoin/nodes/config.toml assets/config.toml
+
+FROM scratch as setup_log
+COPY --from=builder-byzcoin /byzcoin/signup.link .
+COPY --from=builder-byzcoin /byzcoin/nodes/config.toml .
+
+FROM lipanski/docker-static-website:latest as web
+#FROM python:latest
```
Comment (PascalinDe) [inline Dockerfile]: please remove that line :)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
-# Byzcoin node
+# Byzcoin node - ARCHIVED
+
+This repo has been archived and is no longer supporter.
```
Comment (PascalinDe) [inline README.md]:
```suggestion
This repo has been archived and is no longer supported.
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+To run the demo, download the repo and run the `docker-compose.yaml` file:
+
+```bash
+git clone https://github.com/c4dt/byzcoin
+cd byzcoin
+docker compose up -d
+open http://localhost:8080
+```
+
+Then point your browser to http://localhost:8080
```
Comment (PascalinDe) [inline README.md]:
```suggestion
Then point your browser to http://localhost:8080.
```

File: TODO.md [hunk 5]
```diff
@@ -1,2 +1,6 @@
-- make conode.go so that it works with the environment variables
-- fix status so `status --host https://test.com:7771` works
+- fix double sign up
```
Comment (PascalinDe) [inline TODO.md]: shouldn't these be issues?

File: archive/index.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+        <a href="https://factory.c4dt.org/showcase/omniledger/presentation">ByzCoin</a> paper from the
+        <a href="https://dedis.epfl.ch">DEDIS</a> lab at EPFL.
+        It has been supported and extended by the <a href="https://c4dt.epfl.ch">C4DT</a> and is now available
+        as a standalone, archived demo that can be run on your computer.
+    </p>
+    <p>
+        You can find more information on our blog post here: <a href="https://c4dt.epfl.ch/article/omniledger-2/">OmniLedger</a>.
+    </p>
+    <p>
+        When you typed <code>docker compose up</code>, four nodes started communicating with each other
```
Comment (PascalinDe) [inline archive/index.html]:
```suggestion
        When you type <code>docker compose up</code>, four nodes start communicating with each other
```

File: archive/index.html [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+                <p class="card-text">This service allows administration of a user credential on the blockchain.
+                    You can read more in the blog post about
+                    <a href="https://c4dt.epfl.ch/article/dynacred/">Account Management in Omniledger</a>.
+                    Internally we used it to connect to other services, like our matrix instance for internal
+                    discussions: <a href="https://c4dt.epfl.ch/article/cas-for-omniledger/">CAS Login using
+                        OmniLedger</a>.
+                    Connecting mobile devices was a bit more complicated, but today's passwordless authentication
+                    services
+                    also involve scanning of QRCodes, like did our solution to
+                    <a href="https://c4dt.epfl.ch/article/matrix-on-mobile/">Login to Matrix on Mobile</a>.
```
Comment (PascalinDe) [inline archive/index.html]:
```suggestion
                    also involve scanning of QRCodes, like our solution to
                    <a href="https://c4dt.epfl.ch/article/matrix-on-mobile/">Login to Matrix on Mobile</a> did.
```

File: TODO.md (hunk 5 above)
Comment (ineiti) [reply TODO.md]: If I work alone on a project I usually add a TODO and write things down there. It is removed now.

**Review by PascalinDe — APPROVED** [review]
works locally, LGTM

**Pull request merged**